scripts/exrrfs_run_gsidiag.sh

```diff
@@ -304,7 +307,54 @@ if [ ${DO_RADDA} == "TRUE" ]; then
        cp ./satbias_pc.out ${satbias_dir}/rrfs.${spinup_or_prod_rrfs}.${YYYYMMDDHH}_satbias_pc
      fi
   fi
+
+fi
+
+#------------------------------------------------------------------------
+# set up local dirs and run run radmon to generate radiance monitor data
+#------------------------------------------------------------------------
+DO_RADMON=TRUE
```

Comment:
@xyzemc, can you move this to "ush/config_default.sh"? A flag should be defined in there first.

Reply:
done

Reply:
@xyzemc, I think you forgot to remove this line. Since you define this in config_default.sh, you don't have to set this here again. Otherwise, this flag will be always turned on in this script. I don't think this is what you want.

Reply:
> @xyzemc, I think you forgot to remove this line. Since you define this in config_default.sh, you don't have to set this here again. Otherwise, this flag will be always turned on in this script. I don't think this is what you want.

Yes, you are right. I will remove it.